src/main/java/io/mslm/lib/Lib.java

```diff
@@ -0,0 +1,94 @@
+package io.mslm.lib;
+
+import io.mslm.emailverify.SingleVerifyResp;
+import okhttp3.*;
+
+import com.google.gson.Gson;
+import java.net.*;
+
+import java.util.Map;
+
+public class Lib {
+    public String apiKey;
+    public OkHttpClient http;
+    public URI baseUrl;
+    public String userAgent;
+    public static final String version  = "1.0.0";
+
+    public Lib(){
+        apiKey = "";
+        http = new OkHttpClient();
+        baseUrl = URI.create("https://mslm.io");
+        userAgent = version;
+    }
+
+    public Lib(String apiKey){
+        this.apiKey = apiKey;
+        http = new OkHttpClient();
+        baseUrl = URI.create("https://mslm.io");
+        userAgent = "mslm/java/" + version;
+    }
+
+    public void setHttpClient(OkHttpClient httpClient) {
+        this.http = httpClient;}
+
+    public void setBaseUrl(String baseUrlStr) throws Exception {
+        this.baseUrl = new URI(baseUrlStr);
+    }
+
+    public void setUserAgent(String userAgent) {
+        this.userAgent = userAgent;
+    }
+
+    public void setApiKey(String apiKey) {
+        this.apiKey = apiKey;
+    }
+
+    public URI prepareUrl(String urlPath, Map<String, String> qp, ReqOpts opt) throws Exception {
+        // Put the API key to the query params map.
+        qp.put("apiKey", opt.apiKey);
```

Comment:
We expect all lowercase in the API:

```suggestion
        qp.put("apikey", opt.apiKey);
```